File: C++/linearsearch.py
def find(a,x):
	for i in range (0,len(a)):
		if(a[i] == x):
			return i
	return -1

n = int(input())
x = int(input())
a = []

for i in range n:
	a.append(int(input()))

ans = find(a,x)

if(ans==-1):
	print("UNABLE TO FIND")
else:
	print("IS AT %d POSITION"%(ans))

File: C++/linearsearch.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){

	int n,x;
	cin >> n >> x;

	int a[n], p = -1;
	for(int i=0;i<n;i++){
		if(a[i] == x){
			p = i;
			break;
		}
	}

	if(p==-1){
		cout << "Unable to find " << x << endl;
	}else{
		cout << x << " is at position " << p << endl;
	}

	return 0;
}